tpl/template_funcs.go

```diff
@@ -1803,7 +1803,7 @@ func htmlUnescape(in interface{}) (string, error) {
 
 func init() {
 	funcMap = template.FuncMap{
-		"absURL":       func(a string) template.HTML { return template.HTML(helpers.AbsURL(a)) },
+		"absURL":       func(a interface{}) template.HTML { return template.HTML(helpers.AbsURL(cast.ToString(a))) },
```

Comment:
He learned from the best. :wink:  Do the changes in https://github.com/spf13/hugo/commit/3a2a4c3b07f82527765cbeaa33f53ab7e2314491 need errors returned, too?